backend/src/models/User.ts

```diff
@@ -36,6 +64,11 @@ UserSchema.methods.isValidPassword = async function (
   return compare;
 };
 
-const User = mongoose.model<User, UserModel>("user", UserSchema);
+// TODO: should we inherit interfaces defined in types.ts?
```

Comment:
not needed rn but as we add values to this stuff yea you would need to but in the schema instead, like of the userschema imports the user stuff

Reply:
deleted them for now